lib/index.js

```diff
@@ -17,25 +21,25 @@ for(const module of contextModules) {
   const {contexts, constants: contextConstants} = require(module);
   for(const c in contextConstants) {
     const contextUrl = contextConstants[c];
-    bedrockConstants[c] = contextUrl;
-    bedrockConstants.CONTEXTS[contextUrl] = contexts.get(contextUrl);
+    dlc.urls[c] = contextUrl;
+    dlc.documents.set(contextUrl, contexts.get(contextUrl));
   }
 }
 
-bedrockConstants.TEST_CONTEXT_V1_URL = 'https://w3id.org/test/v1';
-bedrockConstants.CONTEXTS[bedrockConstants.TEST_CONTEXT_V1_URL] = JSON.parse(
+dlc.urls.TEST_CONTEXT_V1_URL = 'https://w3id.org/test/v1';
+dlc.documents.set(dlc.urls.TEST_CONTEXT_V1_URL, JSON.parse(
   fs.readFileSync(
     path.join(__dirname, '../contexts/test-v1.jsonld'),
-    {encoding: 'utf8'}));
+    {encoding: 'utf8'})));
 
-bedrockConstants.IDENTITY_CONTEXT_V1_URL = 'https://w3id.org/identity/v1';
-bedrockConstants.CONTEXTS[bedrockConstants.IDENTITY_CONTEXT_V1_URL] =
-  JSON.parse(fs.readFileSync(
+dlc.urls.IDENTITY_CONTEXT_V1_URL = 'https://w3id.org/identity/v1';
```

Comment:
This context needs to go away ... is now an appropriate time for that?

Reply:
Seems like it to me, I'll take it out.

Reply:
Gone.